### DISCUSSION TITLE
SpotX for Android

### DISCUSSION THREAD

**Discussion by MrEddie7** [SpotX-Official/SpotX#662] [Ideas]
Would it be possible to make an Android/IOS version of SpotX in the future? Since most people use it while on the go, it would soon reach more users for the platform.

**Comment by amd64fox** [comment]
not in plans

**Reply by Kitsune-Rei** [reply]
xmanager doesn't work anymore. Since the last months of 2024 spotify patches have been problematic. is there a new or other method for android?

**Reply by Bogdan451** [reply]
> xmanager doesn't work anymore. Since the last months of 2024 spotify patches have been problematic. is there a new or other method for android?

I agree with you. I'm have these problems too.

**Reply by Kitsune-Rei** [reply]
I installed the standard Spotify patch on Xmanager and it worked very well. I suggest you give it a try again.

**Reply by TheParadoxBox** [reply]
xManager works again as of whenever they rebased to Spotify ReVanced. Using the regular ReVanced version is also an option; it gives you a bit more control of features (and you can re-patch whenever you want), but xManager is a little more convenient since it distributes pre-patched APKs.

**Reply by aayankc0210** [reply]
As of Early Feb 2026, none of these work. All of these work arounds face the same problem. The tracks skip automatically and repetitively.

**Reply by TheParadoxBox** [reply]
It's Jan 28. Bro is living in the future

Also I'm very impressed by the critical thinking skills of people thumbs-downing months-old testimonies. It's true, ReVanced doesn't work anymore. I don't know if they're working on a fix; I ended up buying Spotify Premium since I have a little more money now.

**Reply by aayankc0210** [reply]
My bad man, didn't write it correctly. But i did get xmanager working. If you get the latestt version of Xmanager, yo see "Still Kickin". Don't know if they're mocking Spotify but whatever it is. its working for the moment

**Comment by Bogdan451** [comment]
> not in plans
> 
> Use [xManager](https://t.me/xManagerSupport/138498) for android 
> and [EeveeSpotify](https://github.com/whoeevee/EeveeSpotify) for ios

Can i trust telegram channel where can i download apk file for android?

**Reply by Bogdan451** [reply]
I have tried to download, but in spotify, I saw issues. All songs, I can't listen, they are just hide, ALL. And then I have tried to download other file(apk), but again this problem. (Sorry for my English, I learning it(I from Russia))

**Comment by cisko99za** [comment]
really useless request. search on web. it's full of mod apk for android ...
for win desktop app ONLY wonderful spotX

**Reply by Kitsune-Rei** [reply]
I'm not going to ask you what I want. Most modified Android applications only work reliably for one or two days. I asked the community if there is an application that works as reliably as SpotX. If you don't know the answer, please don't write it.

**Reply by 13ushido** [reply]
I'm having the same issue. Didn't want to go the xmanager/revanced route, but that seems like the only way nowadays

**Reply by fmdkcl4e** [reply]
> really useless request. search on web. it's full of mod apk for android ... for win desktop app ONLY wonderful spotX

xmanager and other MODS dont work

**Reply by GLXY-ARCANE** [reply]
So for Android you need to use Antisplit M and ReVanced, but every time Spotify updates, which feels like once a day, you have to re-patch it and hope it works. As of recently it's been more hit and miss on working

**Reply by BaDumTssN** [reply]
None working, tried AntiSplit and Revanced which I have been using for months now, it's now working for at best a day or two, and it takes long for Revanced to patch Spotify.

**Comment by amd64fox** [comment]
The ReVanced devs are testing a new patch and will roll it out to testers in the xManager ds soon. 

Please be patient.

**Reply by 13ushido** [reply]
ds as in Discord?

**Reply by amd64fox** [reply]
yeah

**Reply by Rinary1** [reply]
> yeah

Currently revanced manager taked dmca from spotify and they removing their old patches(which have a bug so you can't use it on new version without auto akipping songs) and stopped developing updates. Idk is there another ways to just open some qol functions without buying spotify premium.

**Reply by winkmod-apk** [reply]
SpotX for Android is a lightweight and efficient tool that enhances the streaming experience. Platforms like [winkmodapk](https://www.winkmodapk.com.co) provide useful discussions and updates for users interested in such apps.
